pkgpanda/__init__.py

```diff
@@ -710,6 +732,12 @@ def symlink_all(src, dest):
 
             symlink_tree(src, dest)
 
+        def copy_all(src, dest):
+            if not os.path.isdir(src):
+                return
```

Comment:
should this return silently? wouldnt this be a usage error?

Reply:
Possibly. My intention here is to duplicate the semantics of `symlink_all()` defined just before this one, which I didn't want to change as part of this PR.